blockchains/avalanche/assets/0xdB25f211AB05b1c97D595516F45794528a807ad8/info.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
note sure why this `https://snowtrace.io/address/0xC891EB4cbdEFf6e073e859e987815Ed1505c2ACD` does not match `blockchains/avalanche/assets/0xdB25f211AB05b1c97D595516F45794528a807ad8` ?

Reply:
Ah thanks for the catch, this is EURS address on mainnet where I copied the folder from. I forgot to rename the folder itself, my bad!

Reply:
Fixed!